src/movement/test/test_thruster_maestro.cpp

```diff
@@ -0,0 +1,146 @@
+//This file is designed to test the serial_subscriber.cpp example, which
+//receives messages of the robosub/thruster type, and then sends the exact
+//received value through a serial port. This example sends a single thruster
+//message with several thrusters inside, and then listens on a serial port to
+//confirm that the received values are the same as the sent values.
+#include <gtest/gtest.h>
+#include "ros/ros.h"
+#include "robosub/thruster.h"
+#include "utility/serial.hpp"
+#include "utility/test_tools.hpp"
+#include <vector>
+#include <string>
+
+rs::Serial mSerial;
+ros::Publisher pub;
+double byte_check(uint8_t byte2, uint8_t byte3);
+std::vector<uint8_t> channels;
+double max_speed = 0.6;
+
+//insert tests here
+TEST(Maestro, basicTest)
+{
+    //allocate an array of bytes that will be used to receive serial data from
+    //the serial port. I use the arbitrarily large size of
+    uint8_t maestro_data[256] = {0};
```

Comment:
I had the same question in the sabertooth test. Why is this an arbitrary number? Isn't this a guaranteed size given correct responses?